dulwich/porcelain.py

```diff
@@ -917,9 +917,9 @@ def update_refs(refs):
             errstream.write(
                 b"Push to " + remote_location_bytes + b" successful.\n")
         except (UpdateRefsError, SendPackError) as e:
+            message, = e.args
```

Comment:
Handling for UpdateRefsError and SendPackError probably needs to be split out here.

UpdateRefsError has more than 1 element in e.args, and e.args[0] is a unicode string.